src/Parser.js

```diff
@@ -354,7 +356,7 @@ Parser.prototype._parseRepeat = function() {
 };
 
 var INPUTS_OUTPUTS_COMMANDS = ['ensure', 'require', 'input', 'output'];
-var STATEMENT_COMMANDS = ['state', 'print', 'return'];
+var STATEMENT_COMMANDS = ['state', 'print', 'return', 'break', 'continue'];
 Parser.prototype._parseCommand = function(acceptCommands) {
```

Comment:
This function is called for all `STATEMENT_COMMANDS` at line 241.

However, the behavior of `break` and `continue` commands is slightly different from `print`, `state` and `return`. `break` and `continue` do not accept any arguments -- they should not have `<open-text>` after them.

In this function, line 365 inserts an `<open-text>` child node for all the statement commands, but that shouldn't happen for `break` and `continue`.

Reply:
Yes, thought it works without `<open-text>` after `\BREAK` and `\CONTINUE`, we also should treat them as new type of commands. I update the code, please check whether it's a better way to do so.